libafl_frida/src/coverage_rt.rs

```diff
@@ -51,8 +75,65 @@ impl FridaRuntime for CoverageRuntime {
 
     fn post_exec<I: libafl::inputs::Input + libafl::inputs::HasTargetBytes>(
         &mut self,
-        _input: &I,
+        input: &I,
     ) -> Result<(), libafl::Error> {
+        if self.save_dr_cov {
```

Comment:
Why not only write dr_cov data for new interesting testcases/inside a feedback?

Reply:
The existing DrCovRuntime writes after each run, however, due to the way it is implemented the files are written only if new blocks have been executed. The idea behind writing for each input is that you can see where your fuzzing time is spent.